pages/successstories/index.tsx

```diff
@@ -1,68 +1,23 @@
 import Head from 'next/head'
```

Comment:
![Screen Shot 2021-03-22 at 3 25 09 PM](https://user-images.githubusercontent.com/72828520/112066029-e3af5680-8b22-11eb-86ae-e3682e4672e5.png)
![Screen Shot 2021-03-22 at 3 25 16 PM](https://user-images.githubusercontent.com/72828520/112066035-e6aa4700-8b22-11eb-9b54-a2248cba035d.png)
![Screen Shot 2021-03-22 at 3 25 23 PM](https://user-images.githubusercontent.com/72828520/112066038-e742dd80-8b22-11eb-985f-391a8e07b57e.png)

i think that we could use the width of pictures and layout as displayed on mobile for all the views. what do you think @vsauceda-cfa ?